sandbox-backend/src/main/resources/db/migration/V0000000__initial.sql

```diff
@@ -91,10 +91,10 @@ CREATE TABLE employee_discount
 
 CREATE TABLE review
 (
-    id          INT      NOT NULL AUTO_INCREMENT PRIMARY KEY,
-    rate        INT      NOT NULL,
-    comment     VARCHAR(500),
-    date        DATETIME NOT NULL,
+    id         INT      NOT NULL AUTO_INCREMENT PRIMARY KEY,
```

Comment:
Please revert all changes in the initial script

Reply:
done.